arch-snapshot/arch.md

```diff
@@ -13,37 +13,81 @@ implement the needed functionalities.
 
 The aim is not to dive deep into every integration, but to capture architectural
 decisions and their consequences.
-# Functional view
+
+# Framework Concept
+
 This section describes what comprises Golem network, namely the actors,
 technical artifacts and activities they actors may perform on those activities.
 The objective of this paragraph is to tie together all the terms and provide a
 very high level description of what they are.
+
 ## What Golem is
 ## What Golem is not
 ## Actors
 This section describes the actors using Golem Network and their role in the
 system.
 ### Providers
 ### Requesters
+
+note: The requestor is what talks directly to the golem node. it could be, for example, a rendering portal. and user is the person who uses this portal. in payment scenarios there are advantages when these roles are separated.
```

Comment:
I tried to split actors more:
https://github.com/golemfactory/golem-architecture/pull/98/commits/cb202161a99128601181681ce8e17588749d75b6